server/middleware/rest.js

```diff
@@ -28,6 +28,7 @@ function rest() {
 
   return function restApiHandler(req, res, next) {
     var app = req.app;
+    var registry = app.registry || loopback.registry;
```

Comment:
Please don't ask users of `app.registry` to handle the case when the per-app registries are not enabled. `app.registry` should be always defined and point to `loopback.registry` when a global registry is used.